docs/content_management/block_reference.md

```diff
@@ -87,6 +88,25 @@ On the **Properties** tab, set values in the following fields:
 
 On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.
 
+## Campaign block
+
+Displays campaign created and configured in Ibexa Engage.
+On the **Properties** tab, set values in the following fields:
+
+- **Name** – Enter a name for the campaign.
+- **Campaign** – Select campaign to display on the Landing Page.
```

Comment:
@radzyu can we mention which campaigns are visible in the list?

Reply:
It's mentioned in the "Campaign block" section: _"This list includes all campaigns available on user's Qualifio account which are active or scheduled to launch in the future."_
in my opinion there is no need to go into details here on the Block reference page.

Reply:
@julitafalcondusza I don't see this mention here in the section: https://ez-systems-developer-documentation--296.com.readthedocs.build/projects/userguide/en/296/content_management/block_reference/#campaign-block